src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -437,15 +455,15 @@ protected function doRequestInProcess($request)
             foreach ($deprecations ? unserialize($deprecations) : [] as $deprecation) {
                 if ($deprecation[0]) {
                     // unsilenced on purpose
-                    trigger_error($deprecation[1], \E_USER_DEPRECATED);
+                    @trigger_error($deprecation[1], \E_USER_DEPRECATED);
```

Comment:
```suggestion
                    trigger_error($deprecation[1], \E_USER_DEPRECATED);
```
The comment above that line looks trustworthy.

Reply:
Oh sorry this changes where suggested by the fabbot.io maybe something need to be added there that it doesn't happen again.

Reply:
/cc @fabpot : https://fabbot.io/report/symfony/symfony/38596/cc1415f58e1292de7849f7a8ffad222ab74b9d95

Reply:
Yes, it's a false positive.